### DISCUSSION TITLE
what's the meaning of `cumulative history of a given directory (putback-log) also in RSS format `?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [oracle/opengrok#4177] [Q&A] [answered]
When I read this document page: [Fearture](https://github.com/oracle/opengrok/wiki/Features)
> cumulative history of a given directory (putback-log) also in RSS format 

what's the meaning of this sentence?

**Comment by vladak** [comment, marked as answer]
If you display history for a directory, there is a tiny RSS feed link near the bottom of the page, e.g. `https://exmple.com/source/rss/opengrok-master/distribution/`. This is a RSS feed with the history for this directory. Not sure it is used much these days.
